tf2/include/tf2/convert.h

```diff
@@ -32,10 +32,14 @@
 #ifndef TF2_CONVERT_H
 #define TF2_CONVERT_H
 
+#include <builtin_interfaces/msg/time.hpp>
+
+// TODO(tfoote) 
+#error This has not been converted to ROS2 and will not compile
```

Comment:
This is currently commented out of the build as the datatype conversions packages were not included. This is how you interact with them. You have to use either the message datatypes or the tf2 native datatypes.